src/main/java/org/apache/gossip/manager/GossipCore.java

```diff
@@ -93,8 +77,15 @@ public GossipCore(GossipManager manager, MetricRegistry metrics){
     messageSerdeException = metrics.meter(MESSAGE_SERDE_EXCEPTION);
     tranmissionException = metrics.meter(MESSAGE_TRANSMISSION_EXCEPTION);
     tranmissionSuccess = metrics.meter(MESSAGE_TRANSMISSION_SUCCESS);
+
+    handlers = new HashMap<>();
+    handlers.put(Response.class, new ResponseHandler());
+    handlers.put(ShutdownMessage.class, new ShutdownMessageHandler());
+    handlers.put(GossipDataMessage.class, new GossipDataMessageHandler());
+    handlers.put(SharedGossipDataMessage.class, new SharedGossipDataMessageHandler());
+    handlers.put(ActiveGossipMessage.class, new ActiveGossipMessageHandler());
```

Comment:
This should move to ManagerBuilder class provide users sensible defaults but give the ability to set the list themselves.